release_notes/topics/16_0_0.adoc

```diff
@@ -0,0 +1,7 @@
+= Highlights
+
+== OpenID Connect Front-Channel Logout Support
+
+{project_name} now supports https://openid.net/specs/openid-connect-frontchannel-1_0.html[OpenID Connect Front-Channel Logout 1.0].
```

Comment:
Small note: Do we want to highlight the contributor, who initiated this effort and adding his github link? See other release notes for various features contributed by community.